wp-bootstrap/wp-bootstrap-navwalker
# Remove SiteNavigationElement from menu items

Remove ` itemscope="itemscope" itemtype="https://www.schema.org/SiteNavigationElement"` as it is not functional, unnecessary (out of scope) and causes conflict with SEO plugins with no way to disable.

Fixes #440 

#### Changes proposed in this Pull Request:

Avoid conflicting schema markup

#### Testing instructions:

Load URL page with this plugin here: https://search.google.com/structured-data/testing-tool/ to see if  SiteNavigationElement is present.


#### Proposed changelog entry for your changes:

Remove conflicting SiteNavigationElement structured data / schema markup
